static/js/topic_page/block.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for rendering a block.
+ */
+
+import React from "react";
+
+import { randDomId } from "../shared/util";
+import { StatVarMetadata } from "../types/stat_var";
+import { LineTile } from "./line_tile";
+import { MapTile } from "./map_tile";
+import { RankingTile } from "./ranking_tile";
+import { Tile } from "./tile";
+
+export interface BlockPropType {
```

Comment:
from the proto, it looks like a block can also contain a description, probably want to include a field for that here in the BlockPropType

Reply:
Done